app/static/css/style.css

```diff
@@ -11,7 +11,7 @@
 
 }
 /*---Firefox Bug Fix --*/
-.carousel-item {
+.ousel-item {
```

Comment:
Is this class needed? as I don't think it's needed

Reply:
This was default css when implementing bootstrap. I don't believe it makes a difference but was recommended to keep

Reply:
I also left the carousel text the same size to keep consistency with the rest of the page as no other text resizes. I can implement it to change size at certain page width if you preferred?

Reply:
This was default css when implementing bootstrap. I don't believe it makes a difference but was recommended to keep.
I have for now fixed the missing characters

Reply:
I also left the carousel text the same size to keep consistency with the rest of the page as no other text resizes. I can implement it to change size at certain page width if you preferred?

Reply:
I'll check with others whether they expect the image to resize but for now lets leave it as it is